### ISSUE TITLE
Consider treating Notebooks in remote repositories like they are in untrusted workspaces

### ISSUE THREAD

**Issue by SiddhanthUnnithan** [microsoft/vscode-jupyter#6949]
**TL;DR** The Notebook experience for remote repositories should be consistent with untrusted workspaces until running cells is supported. At a high-level, the extension should omit all functionality related to the Python extension and kernel usage, such as the interactive window and Notebook export.

**Activation**
Currently, if the user does not have the Python extension installed when trying to use Jupyter they're prompted to install it. As the Python extension is disabled in remote repositories, we should omit showing the install prompt.

![image](https://user-images.githubusercontent.com/6413608/127923117-fe2b1a6d-454d-4b2f-8ff5-fba73c986fa6.png)

**Interactive Window**
Within a remote repository, users can create an interactive window using either the command palette or by adding the right signatures to a Python script. The window that is opened displays a toast that a kernel is being connected to, however the connection is never established and the created panel remains blank. Until connecting to kernels and running code is officially supported, users should not be shown:
1. The "Create Interactive Window" command from the palette.
2. The Codelens option in Python scripts when the "#%%" signature is added.
3. The file context menu options for working with the interactive window.

**Exports**
Users are shown the regular export options however clicking on any one of them will result in a continuously-running toast. Export options should be omitted from the Notebook toolbar within a remote repository until Python and kernel functionality is included.

**Variable Explorer and Notebook Toolbar**
Users cannot make use of the variable explorer, restart, or interrupt options with cell running disabled. All three of those options should be completely removed from the toolbar, as opposed to being shown as disabled in the case of the kernel-interaction options.

![image](https://user-images.githubusercontent.com/6413608/127923135-710d55bc-cbfd-402d-9944-72cff9f075ee.png)

**Limited Functionality Messaging**
It should be made clear to users that they have limited Notebook functionality in a remote repository. When accessing the extension from the marketplace, the user should be shown (via a hover-over option) that "Execution of cells in interactive window and notebooks is unsupported in a virtual workspace"..

**Comment by greazer** [comment]
Based  on some discussion and experimentation, it seems that disabling the Jupyter extension for remote repositories continues to offer a very basic visualization support for notebooks, due to VS Code's built-in ipynb serializer and renderers.

**Comment by DonJayamanne** [comment]
Closing this issue for now as there haven't been enough upvodes on this feature request
